oauth/bin/www

```diff
@@ -21,8 +21,8 @@ app.set("port", port);
  * Create HTTP server.
  */
 const credentials = {
-  cert: fs.readFileSync(path.join(__dirname, "./cert.pem")),
-  key: fs.readFileSync(path.join(__dirname, "./key.pem")),
+  cert: fs.readFileSync(path.join(__dirname, "../localhost.pem")),
+  key: fs.readFileSync(path.join(__dirname, "../localhost-key.pem")),
 };
```

Comment:
I always find it useful to, in samples, add something like

```js
if (!fs.existsSync(certPath) || !fs.existsSync(keyPath)) {
  // don't "throw" an error to make it clear it's not an "internal" error
  console.error('Please generate certificates using mkcert, first. See README.md for more details.')
  process.exit(1); 
}
```

to avoid the "developer journey" of not reading the `README` and "getting that strange error when trying to run it" 😉